cmd/expired-authz-purger/main.go

```diff
@@ -117,6 +126,46 @@ func (p *expiredAuthzPurger) getWork(work chan string, query string, initialID s
 	return lastID, count, nil
 }
 
+// deleteAuthorizations reads from the work channel and deletes each authorization
+// from either the pendingAuthorization or authz tables. If maxDPS is more than 0
+// it will throttle the number of DELETE statements it generates to the passed rate.
+func (p *expiredAuthzPurger) deleteAuthorizations(work chan string, maxDPS int, parallelism int, table string, checkpointFile string) {
+	wg := new(sync.WaitGroup)
+	deleted := int64(0)
+	var ticker *time.Ticker
+	if maxDPS > 0 {
+		minDur := time.Duration((float64(parallelism)/float64(maxDPS))*1000) * time.Millisecond
```

Comment:
Sorry, I should have caught this earlier: Since all the workers pull from the same ticker, there's no need to multiply by parallelism.

Also, regarding the truncation thing, what do you think of:

```
time.Duration(float64(time.Second)/float64(maxDPS))
```

I think that winds up being clearer; otherwise to a casual reader it raises the question "is this config specified in second or millseconds" (it's actually in Hz :-D)